src/lenskit/data/builder.py

```diff
@@ -359,6 +366,8 @@ def add_relationships(
                 Whether this is an interaction relationship or not; can be
                 ``"default"`` to indicate this is the default interaction
                 relationship.
+            remove_duplicates:
+                If ``True``, exact duplicate interactions will be removed.
```

Comment:
Need to update this to more clearly reflect the full (intended) behavior, and probably also change the name to be clearer. I think it would be better to call it `remove_repeats`, and it has 3 options (as currently allowed by the type):

- `False`: repeats are not removed
- `True`: repeats are removed
- `"exact"`: only exact duplicates are removed